README

```diff
@@ -9,29 +9,70 @@ Directory contents:
 		target built from this directory is libest.a.
     /example	This directory contains sample code for implementing
```

Comment:
If we're going into the file to make it correct, can we change this explanation of the /example directory to,

This directory contains sample code for implementing an application that utilizes the libest library.

Need to make it absolutely clear that the logic that is in the /example directory does no implement EST.